src/page-modules/contact/formInputValidator.ts

```diff
@@ -0,0 +1,302 @@
+import { PageText, TranslatedString } from '@atb/translations';
+import { string } from 'zod';
+
+export type InputErrorMessages = {
+  [key: string]: TranslatedString[];
+};
+
+// Utility function to add error messages
+const addErrorMessage = (
+  context: any,
+  field: string,
+  errorMessages: InputErrorMessages,
+  errorMessage: TranslatedString,
+) => {
+  if (!context[field]) {
+    if (!errorMessages[field]) {
+      errorMessages[field] = [];
+    }
+    errorMessages[field].push(errorMessage);
+  }
+};
+
+// Helper function to set fields in travel guarantee machine, which only uses on statemachine for all three formes.
+const setFieldsToValidateSpecialCase = (stateSubmitted: string) => {
```

Comment:
A lot of duplicate code here. Could create a "commonFields" and merge

Reply:
True, will do👍